tests/audit_test.py

```diff
@@ -170,15 +171,33 @@ def test_audit_end_to_end(session, users, groups, http_client, base_url, graph):
     assert groupname in [x.group.name for x in open_audits], "group we expect also gets audit"
 
     # pull all the info we need to resolve audits, avoids detached sqlalchemy sessions
-    AuditMember = namedtuple("AuditMember", "am_id, edge_type, edge_id")
-    Audit = namedtuple("Audit", "audit_id, owner_name, group_name, audit_members")
+    # (DetachedInstanceError)
+    MyAuditMemberInfo = NamedTuple(
+        "MyAuditMemberInfo", [("am_id", int), ("edge_type", int), ("edge_id", int)]
```

Comment:
Nit: Preference for `am_id` -> `member_id` for clarity

Reply:
no, member_id will be confusing with the id of the User/Group object. am_id is clearer that its the id of the audit_member.